libs/coin-framework/src/api/types.ts

```diff
@@ -24,14 +24,9 @@ export type Transaction = {
   recipient: string;
   amount: bigint;
   fee: bigint;
-  supplement?: unknown;
-};
+} & Record<string, unknown>; // Field containing dedicated value for each blockchain
```

Comment:
By doing this, we will not isolate dedicate fields into a substructure 🤔 
I think it's a shame :(

Reply:
@sprohaszka-ledger this data structure uses the flattened form in lama-adapter API, then transformed with `supplement` just for here, then back to flattened form internally, does this really bring any value ?

Suggestion: also add a `type` field and turn this into a tagged union so we can have an exact type for each family

Reply:
discussed with Stéphane and Jérôme => agreed to keep current API until we have a better idea of how we want to handle this with tron integration